scripts/openstack-quickstart-demosetup

```diff
@@ -264,7 +264,8 @@ EODASHDB
 SESSION_COOKIE_SECURE = CSRF_COOKIE_SECURE = USE_SSL
 EOSEC
     # sync dashboard DB "after" the database is created
-    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' syncdb --noinput"
+    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' makemigrations"
```

Comment:
I'm willing to settle for omitting `makemigration`, but `migrate` is definitely needed since it creates a bunch of things :-)

I'll give this another spin without makemigration, and see where that gets me...